save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ContestView.kt

```diff
@@ -103,6 +107,9 @@ class ContestView : AbstractView<ContestViewProps, ContestViewState>(false) {
 
     override fun componentDidMount() {
         super.componentDidMount()
+        scope.launch {
```

Comment:
Why do you need to launch a coroutine here?

Reply:
You are not doing any request

Reply:
Sorry, deleted this